src/main/java/games/strategy/triplea/UrlConstants.java

```diff
@@ -11,13 +11,13 @@
  * </pre></code>
  */
 public enum UrlConstants {
-  GITHUB_ISSUES("http://github.com/triplea-game/triplea/issues/new"),
-  RULE_BOOK("http://triplea-game.github.io/files/TripleA_RuleBook.pdf"),
+  GITHUB_ISSUES("https://github.com/triplea-game/triplea/issues/new"),
+  RULE_BOOK("https://triplea-game.github.io/files/TripleA_RuleBook.pdf"),
```

Comment:
triplea-game.github.io is just an alias for triplea-game.org 
It redirects automatically...

Reply:
Done in 47d05ab.